server/pages/index.jade

```diff
@@ -110,25 +110,45 @@ html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width'
 				}
 			})();
 
-		if user
+		if supportUser
+			script.
+				window.supportUser = {
+					readyScripts: []
+				};
+		if user && !supportUser
 			script(type='text/javascript')!='var currentUser = ' + sanitize.jsonStringifyForHtml( user )
 		if app
 			script(type='text/javascript')!='var app = ' + sanitize.jsonStringifyForHtml( app )
-		if initialReduxState
+		if initialReduxState && !supportUser
```

Comment:
Yes, I'm curious why we need `supportUser` checks on server side pages at all. Seems to add quite a bit of complexity.